mediachain/indexer/mc_ingest.py

```diff
@@ -340,7 +340,7 @@ def tail_blockchain(via_cli = False):
 
     cur = SimpleClient()
     
-    for art in cur.read_artefacts():
+    for art in cur.get_artefacts():
```

Comment:
This (and a lot of others) gets caught by a simple lint. @autoencoder let's get this linting on travis?